### ISSUE TITLE
Needs Update For v50

### ISSUE THREAD

**Issue by nephatrine** [nyakowint/EverythingGoBoom-LC#19]
Getting these errors in v50.

```
[22:16:48.5605862] [Error  : Unity Log] NullReferenceException: Null method for nwnt.EverythingCanDie
Stack trace:
HarmonyLib.PatchProcessor.Patch () (at <474744d65d8e460fa08cd5fd82b5d65f>:IL_001D)
HarmonyLib.Harmony.Patch (System.Reflection.MethodBase original, HarmonyLib.HarmonyMethod prefix, HarmonyLib.HarmonyMethod postfix, HarmonyLib.HarmonyMethod transpiler, HarmonyLib.HarmonyMethod finalizer, HarmonyLib.HarmonyMethod ilmanipulator) (at <474744d65d8e460fa08cd5fd82b5d65f>:IL_0031)
EverythingCanDie.Plugin.CreateHarmonyPatch (HarmonyLib.Harmony harmony, System.Type typeToPatch, System.String methodToPatch, System.Type[] parameters, System.Type patchType, System.String patchMethod, System.Boolean isPrefix) (at <585de499163344adbff9d213cc330589>:IL_008B)
EverythingCanDie.Plugin.Awake () (at <585de499163344adbff9d213cc330589>:IL_0146)
UnityEngine.GameObject:AddComponent(Type)
BepInEx.Bootstrap.Chainloader:Start()
UnityEngine.Rendering.HighDefinition.HDRenderPipelineAsset:OnEnable()

[22:16:48.5605862] [Error  : Unity Log] NullReferenceException: Null method for nwnt.EverythingCanDie
Stack trace:
HarmonyLib.PatchProcessor.Patch () (at <474744d65d8e460fa08cd5fd82b5d65f>:IL_001D)
HarmonyLib.Harmony.Patch (System.Reflection.MethodBase original, HarmonyLib.HarmonyMethod prefix, HarmonyLib.HarmonyMethod postfix, HarmonyLib.HarmonyMethod transpiler, HarmonyLib.HarmonyMethod finalizer, HarmonyLib.HarmonyMethod ilmanipulator) (at <474744d65d8e460fa08cd5fd82b5d65f>:IL_0031)
EverythingCanDie.Plugin.CreateHarmonyPatch (HarmonyLib.Harmony harmony, System.Type typeToPatch, System.String methodToPatch, System.Type[] parameters, System.Type patchType, System.String patchMethod, System.Boolean isPrefix) (at <585de499163344adbff9d213cc330589>:IL_008B)
EverythingCanDie.Plugin.Awake () (at <585de499163344adbff9d213cc330589>:IL_0146)
UnityEngine.GameObject:AddComponent(Type)
BepInEx.Bootstrap.Chainloader:Start()
UnityEngine.Rendering.HighDefinition.HDRenderPipelineAsset:OnEnable()
```

**Comment by nyakowint** [comment]
Will hopefully get to it soon

**Comment by nyakowint** [comment]
Fixed 0f48485285550beb16267facc273b73b02f25cda
